config/locales/spotlight.en.yml

```diff
@@ -1,4 +1,9 @@
 en:
+  time:
+    formats:
+      date:
+        formats:
+          default: '%Y-%m-%d %l:%M%P'
```

Comment:
The big advantage of a built-in format is you'll get existing localizations for free. It's just a question whether that format is so important or semantically different that it deserves special handling.